js/src/plugins/ajaxec.plugin.js

```diff
@@ -32,7 +32,7 @@ export default class ajaxec extends atkPlugin {
         const data = this.settings.uri_options ? this.settings.uri_options : {};
 
         // retrieve param from url.
-        let urlParam = $.atkGetQueryParam(this.settings.uri);
+        let urlParam = $.atkGetQueryParam(decodeURIComponent(this.settings.uri));
```

Comment:
Will this work if `this.settings.uri` = `http://x/y%3Fq`?

Reply:
Why do you think it will not work?

Reply:
`this.settings.uri` seems to be that it can contain URL in standard format like `http://x/y?q` or `http://x/y%3Fq` (URL with path part = `/y%3Fq`, i.e. totaly different URL).

If so, then decoding `http://x/y%3Fq` to `http://x/y?q` is wrong.

Reply:
I see. This will cover it. https://github.com/atk4/ui/pull/945/commits/4e4eaca3e4475d6389840659fde75cbae1d246f8

Reply:
@ibelar Tested, very good, thanks!